File: src/pages/exercises/function-fibonacci-sequence/_codes/php/response/src/fibonacci-sequence.php
<?php

function fibonacci($number)
{
  $a = 0;
  $b = 1;
  $result = "{$a}, {$b}";

  if ($number > 1) {
    while ($number > 2) {
      $temp = $a + $b;
      $a = $b;
      $b = $temp;

      $number--;
      $result .= ", {$temp}";
    }
  } else if ($number === 1) {
    return '0';
  } else {
    return '';
  }

  return $result;
}

File: src/pages/exercises/array-fibonacci-sequence/_codes/python/response/fibonacci_sequence.py
def fibonacciSequence(num: int):
  if (num > 1):
    f0 = 0
    f1 = 1
    sequence = [f0, f1]
    fTemp = None

    for i in range(2, num):
      # add the two previous fibonacci numbers
      fTemp = f0 + f1

      # update the previous fibonacci numbers
      f0 = f1
      f1 = fTemp

      # add the new fibonacci number to the sequence
      sequence.append(fTemp)

    return sequence

  elif (num == 1):
    return [0]

  else:
    return []